### ISSUE TITLE
How to run moatless?

### ISSUE THREAD

**Issue by zdaoguang** [aorwall/moatless-tools#25]
Hi,

Thank you for your excellent work! I would like to request a detailed tutorial on how to run moatless. I am eager to use it but am having difficulty identifying the entry file. Could you please provide guidance on this?

Thank you very much!

**Comment by JensRoland** [comment]
I don't believe there is an entry file as such, but a good starting point is the example Jupyter notebook, which can be run either under Google Colab or locally inside VS Code:

See https://github.com/aorwall/moatless-tools/blob/main/notebooks/00_index_and_run.ipynb

If you wanted to, you could refactor this into a CLI tool, though at this point I think a notebook makes more sense, since the indexing/embedding step isn't persisted/diffed, so in order to run consecutive instructions you'd either have to make it a REPL style CLI tool or implement persistence.

I'm sure this is going to happen in time, but for now, the notebook is good enough.
